HW5/AndriiOliinyk/05_2_Practical tasks_GitHub.py

```diff
@@ -0,0 +1,14 @@
+def fibonacci_to_n(n):
```

Comment:
good job
look at this version of code
n = int(input('enter some number from fibonacci list= '))

fibonacci_numbers = [0, 1]

for i in range(2,n):
        fibonacci_numbers.append(fibonacci_numbers[i-1]+fibonacci_numbers[i-2])
print(f'there are some numbers from fibanacci till that you entered {n},\n {fibonacci_numbers}')